keyboards/idb_60/idb_60.h

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+#include "quantum.h"
+
+inline void _idb_60_caps_led_on(void) {
+    setPinOutput(C5);
+    writePinLow(C5);
+}
+
+inline void _idb_60_esc_led_on(void) {
+    setPinOutput(C4);
+    writePinLow(C4);
+}
+
+inline void _idb_60_caps_led_off(void) {
+    setPinInput(C5);
+    writePinLow(C5);
+}
+
+inline void _idb_60_esc_led_off(void) {
+    setPinInput(C4);
+    writePinLow(C4);
+}
+
```

Comment:
But you'll need to initialize the pins: 

```suggestion

void keyboarb_pre_init_kb(void) {
    setPinOutput(C4);
    setPinOutput(C5);
}
```

Reply:
`keyboard_pre_init_kb` *